doc/Learn/Navigation/HowTo-NavigateInCode.md

```diff
@@ -3,18 +3,21 @@ uid: Uno.Extensions.Navigation.HowToNavigateInCode
 ---
 # How-To: Navigate in Code
 
-This topic walks through controlling Navigation from code, either in the code-behind file of a Page or in the corresponding view model. One of the Navigation objectives was a single navigation construct that applies where ever you choose to write your navigation code.
+This topic walks through controlling Navigation from code, either in the code-behind file of a Page or in the corresponding view model. One of the Navigation objectives was a single navigation construct that applies wherever you choose to write your navigation code.
 
 ## Step-by-steps
 
 > [!IMPORTANT]
 > This guide assumes you used the template wizard or `dotnet new unoapp` to create your solution. If not, it is recommended that you follow the [Creating an application with Uno.Extensions article](xref:Uno.Extensions.HowToGettingStarted) for creating an application from the template.
 
+> [!NOTE]
+> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however it uses the `Mvvm` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Comment:
```suggestion
> This guide uses predefined code created by the Uno Template using the `Recommended` preset, however, it uses the `MVVM` approach for the examples instead of `MVUX` defined in the `Recommended` preset.
```

Reply:
See comment here https://github.com/unoplatform/uno.extensions/pull/2314/files#r1622685604

Reply:
Resolving as it's the same as https://github.com/unoplatform/uno.extensions/pull/2314#discussion_r1622684275